packages/core/src/__tests__/exports.test.ts

```diff
@@ -0,0 +1,12 @@
+import * as components from "../components/index";
+import * as nextComponents from "../next";
+
+describe("exports", () => {
```

Comment:
Ran it on master and then again on the branch to verify that the tests has a correct baseline and all components are exported as expected